src/synced_services/toggl/toggl_synced_service.ts

```diff
@@ -337,6 +338,32 @@ export class TogglTrackSyncedService implements SyncedService {
     return entries;
   }
 
+  async replaceTimeEntryDescription(timeEntry: ServiceTimeEntryObject, tagName: number | string) {
+    let timeEntryFromApi = await this.getTimeEntryById(timeEntry.id, new Date(new Date().setDate(new Date().getDate() - 180)));
+
+    if (timeEntryFromApi === null) {
+       return;
+    }
+
+
+
+    let body =
+      [
+        {'op': 'replace',
+          'path': '/description',
+          'value': timeEntryFromApi != null ? timeEntryFromApi?.text + ` ${tagName}` : `${tagName}`
+        }
+      ]
+    ;
```

Comment:
generovanie body by som tiez pre cistotu kodu preniesol do vlastnej funkcie

Reply:
V tento moment se obavam ze by alepson pro me byl kod oddelenim generovani body mene prehledny.

Kdybychom se do budoucna rozhodli vyuzivat patch na vic veci hodilo by se spise oddelit body a samotne volani do funkce a pak napr. v teto funkci bychom jen volali _updateTimeEntry(op, path, value) pripadne array techto trojic.

Reply:
za mna je to mene prehladne tak, ako to je. Ty si ale aktualnehlavny dev, takze to mozeme ponechat takto.